5-infrastructure/modules/bastion/README.md

```diff
@@ -0,0 +1,97 @@
+# Bastion
+
+## Purpose
+
+Create a Bastion VM using the `terraform-google-modules/bastion-host` module. All values for the module are passed as variables with one exception; the network tag is set to "bastion"
+
+> Reference: [terraform-google-bastion-host](https://github.com/terraform-google-modules/terraform-google-bastion-host)
+
+## Unit Tests
```

Comment:
We do not need module level unit tests as the bastion module is tested

Reply:
READMEs are not updated and were taken directly from non-integrated module

Reply:
Done in 795f245c193e34c5cc48c7ee516910b80e3f09c2